### DISCUSSION TITLE
Session cookies disappearing on page reload (using GraphQL Client)

### DISCUSSION THREAD

**Discussion by byurhannurula** [vercel/next.js#12982] [Help]
Hi, I'm having issues with session cookies.
On localhost everything works perfect but after deploying (next build && next start) on page reload cookies are disappearing.

I'm using GraphQL Client with Next.js on front-end and Node/Express with GraphQ Server on server side. Deployed on same domain tried almost everything.

App itself: http://bank.bbn.codes/
Repo: https://github.com/byurhanbeyzat/bank-app-client

**Comment by brianlovin** [comment]
Can you share any of the server-side code where you are creating and setting the cookie?

**Comment by rafinskipg** [comment]
@byurhanbeyzat  I tried your app and it is working, the cookie stays there when I do a page reload.

**Comment by byurhannurula** [comment]
@brianlovin  @rafinskipg 
I've resolved it with removing next-with-apollo lib

Here is my final code: https://github.com/apollographql/apollo-client/issues/4190#issuecomment-629852792
